src/sentry/options/defaults.py

```diff
@@ -1260,6 +1260,15 @@
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
 
+# Option to remove support for percentiles on a per-use case basis.
+# Add the use case to list to disable percentiles.
```

Comment:
_nit_: This isn't a list, the use case must be an entry with a list value (otherwise causing a crash). Should we make this more clear?

Reply:
Good callout, thanks